### ISSUE TITLE
OpenVAS scanner interrupted at 0%

### ISSUE THREAD

**Issue by Franco-Sparrow** [greenbone/openvas-scanner#758]
O.S: Debian Buster 10.9
CPU: 4 cores
M.RAM: 4 GB
KVM Virtual Machine
GVM version: 21.4.0

All packages were downloaded from Greenbone official repository here in Github and all were built from source, following the procedure from the file attached ("gvm-21.04 (v1.6)")
[gvm-21.04 (v1.6).txt](https://github.com/greenbone/openvas-scanner/files/6642927/gvm-21.04.v1.6.txt)

The scanner is interrupted at 0%. The only error I can see is from "journalctl -f":

```
Jun 11 17:22:30 gvm sudo[4459]:      gvm : TTY=unknown ; PWD=/opt/gvm ; USER=root ; COMMAND=/opt/gvm/sbin/openvas --scan-start 335d82ce-a5d1-4e1c-8c0c-68ff21f79dd2
Jun 11 17:22:30 gvm sudo[4459]: pam_unix(sudo:session): session opened for user root by (uid=0)
Jun 11 17:22:31 gvm python[636]: (process:1760): GLib-WARNING (recursed) **: Invalid UTF-8 passed to g_io_channel_write_chars().
```

I wrote a post in the forum about this issue, but no lucky with the spected feedback.
https://community.greenbone.net/t/openvas-scanner-interrupted-at-0/9386

I have followed the tutorials from:
https://sadsloth.net/post/install-gvm11-src-on-debian/
https://kifarunix.com/install-and-setup-gvm-20-08-on-ubuntu/

Both of them has issues and are incompatible with GVM 21.4.0.

I hope someone could help me here.

Cheers

**Comment by ArnoStiefvater** [comment]
We now have [official installation guides](https://greenbone.github.io/docs/) which can be used for installing GVM.

I'm closing this issue as invalid for now. If this issue about the `GLib-WARNING` persists even after using the official install guide please create a new issue and we will look into it.

**Comment by ArnoStiefvater** [comment]
The `GLib-WARNING` might also already be fixed with greenbone/gvm-libs#499
